ddmrp/models/stock_buffer.py

```diff
@@ -804,6 +804,18 @@ def _compute_main_supplier(self):
             else:
                 rec.main_supplier_id = False
 
+    @api.depends("main_supplier_id", "product_id.seller_ids")
+    def _compute_product_vendor_code(self):
+        for rec in self:
+            if not rec.main_supplier_id:
+                rec.product_vendor_code = False
+                continue
+            supplier_info = rec.product_id.seller_ids.filtered(
```

Comment:
Could you adjust to use the method `_get_product_sellers`?

Reply:
Done